contrib/pg_tracing/sql/select_big.sql

```diff
@@ -0,0 +1,28 @@
+CREATE EXTENSION pg_tracing;
+SELECT pg_tracing_reset();
+
+
+SELECT traces from pg_tracing_info;
+
+/*dddbs='postgres.db',traceparent='00-00000000000000007ed39623a1d7b2d2-7ed39623a1d7b2d2-01'*/ SELECT 1;
```

Comment:
You should be able to do that with a for loop in plpgsql

```
do $$
begin
   for counter in 1..5 loop
	raise notice 'counter: %', counter;
   end loop;
end; $$
```

Reply:
Actually I can't repeat query inside of a procedure at best it will count as one and give something like that :
```

 ?column? 
----------
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
        1
(22 rows)
```